packages/cysync-core/src/pages/MainApp/Settings/Tabs/AppSettings.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from 'react';
+import { Flex, LangDisplay, Toggle, Typography } from '@cypherock/cysync-ui';
+import { SettingsButton, TabItem } from '../components';
+import { selectLanguage, useAppDispatch, useAppSelector } from '~/store';
+import {
+  openChangePasswordDialog,
+  openRemovePasswordDialog,
+  openResetCySyncDialog,
+  openSetPasswordDialog,
+} from '~/actions';
+
+export const AppSettings: React.FC = () => {
+  const { strings } = useAppSelector(selectLanguage);
+  const { item } = strings.settings.tabs.app;
+  const dispatch = useAppDispatch();
+  const [isAnalyticsAndBugReportEnabled, setAnalyticsAndBugReportEnabled] =
+    useState<boolean>(false);
+  const [isAutoUpdateCySyncEnabled, setAutoUpdateCySyncEnabled] =
+    useState<boolean>(false);
+
+  return (
+    <Flex
+      $alignSelf="stretch"
+      direction="column"
+      align="stretch"
+      px={{ def: 3, lg: 5 }}
+      py={{ def: 2, lg: 4 }}
+      gap={32}
+    >
```

Comment:
Since this part is common for all Tabs, it's better if we can keep it in a common component so we don't need to rewrite it.

Reply:
Moved this part to `../Settings/index.tsx` and using react fragment.